syntax = "proto3";
option cc_enable_arenas = true;

package NetProto;

message ReloadLuaScripts
{
	repeated string scripts = 1;
}

message RecreateSceneReq
{
	string scene_name = 1;
}

message RecreateSceneRsp
{
	bool is_succ = 1;
}
